frontend/catalyst/jit.py

```diff
@@ -718,6 +725,14 @@ def _validate_configuration(self):
             )
 
     def _verify_static_argnums(self, args):
+        if not (
+            isinstance(self.compile_options.static_argnums, tuple)
+            and all(isinstance(arg, int) for arg in self.compile_options.static_argnums)
+        ):
+            raise TypeError(
+                "The `static_argnums` argument to `qjit` must be an int or convertable to a"
+                f"tuple of ints, but got value {self.compile_options.static_argnums}"
+            )
```

Comment:
Even more minor, feel free to ignore it.

```suggestion
           msg = ...
           msg += ... # if it s too long to fit in one line
                            # maybe there are other ways to break it up
           raise TypeError(msg)
```

Reply:
I didn't change this. I think I am fine with both ways but I tried to check other cases in codebase to make the code more uniform in style and looks like others are doing the same thing.